package.json

```diff
@@ -35,33 +35,28 @@
     "url": "http://www.bentley.com"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "~5.55.0",
-    "@typescript-eslint/parser": "~5.55.0",
-    "eslint-import-resolver-node": "0.3.4",
-    "eslint-import-resolver-typescript": "^2.7.1",
-    "eslint-plugin-deprecation": "^1.4.1",
-    "eslint-plugin-import": "^2.27.5",
+    "@typescript-eslint/eslint-plugin": "~6.19.1",
+    "@typescript-eslint/parser": "~6.19.1",
+    "eslint-plugin-deprecation": "^2.0.0",
+    "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jam3": "^0.2.3",
-    "eslint-plugin-jsdoc": "^43.2.0",
-    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-jsdoc": "^48.0.3",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-prefer-arrow": "^1.2.3",
-    "eslint-plugin-react": "^7.32.2",
+    "eslint-plugin-react": "^7.33.2",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "workspace-tools": "^0.34.6"
+    "workspace-tools": "^0.36.4"
   },
   "peerDependencies": {
-    "eslint": "^8.36.0",
+    "eslint": "^8.56.0",
     "typescript": "^3.7.0 || ^4.0.0 || ^5.0.0"
   },
   "devDependencies": {
-    "@types/eslint": "~7.2.14",
-    "@types/estree": "~0.0.52",
-    "@types/node": "^18.16.6",
```

Comment:
were not using any node apis?

Reply:
we are but pretty much entire source code (the parts that use the node api at least) is written in javascript and not typescript and there are no JSDoc type directives or anything like that that refer to node api types so @types/node is not needed at the current state of this repo

Reply:
don't remove them if we're using them. Even in JS they provide useful intellisense in LSP editors like vscode.

Reply:
don't remove them if we're using them. Even in JS they provide useful intellisense in LSP editors like vscode.